src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -71,27 +102,6 @@ def get(self, request, project):
             elif rule_type.startswith("action/"):
                 action_list.append(context)
 
-        for install in SentryAppInstallation.get_installed_for_org(project.organization_id):
```

Comment:
I think you extracted this function but forgot to call it.

Reply:
Yep, my mistake -- it shouldn't be called, it was extracted into `node.get_custom_actions()`. See https://github.com/getsentry/sentry/pull/28659